README.md

```diff
@@ -54,6 +54,7 @@ The extension should now be in the project root under `ghost-inspector.ghost-ins
 Please report any issues [on Github](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/issues) or [through our support channel](https://ghostinspector.com/support/).
 
 ## Change Log
+ - 2020-01-27 - `1.0.8`: Updates references in docs from "VSTS" to "Azure DevOps"
```

Comment:
There are 3 spots you will need to updated the version:

 * [`vss-extension.json`](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/blob/master/vss-extension.json#L6)
 * [`run-suite-task/package.json`](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/blob/master/vss-extension.json#L6) (which you've already done)
 * [`run-suite-task/task.json`](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/blob/master/run-suite-task/task.json#L12)